.github/workflows/main.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Why exclude maxos-latest?

Reply:
When upgrading to the ubuntu-24.04 runner, I wanted to know what is currently happening with ubuntu-latest, currently ubuntu-22.04 and whether pillow is also compiled from the sources there. For this investigation the mac-build was irrelevant and therefore I removed it